lib/galaxy/webapps/base/controller.py

```diff
@@ -1577,3 +1577,15 @@ def sort_by_attr(seq, attr):
     intermed = [(getattr(v, attr), i, v) for i, v in enumerate(seq)]
     intermed.sort()
     return [_[-1] for _ in intermed]
+
+
+__all__ = (
+    "HTTPBadRequest",
+    "SharableMixin",
+    "sort_by_attr",
+    "url_for",
```

Comment:
Re-exported, imported from here in `lib/galaxy/webapps/galaxy/api/forms.py` , `lib/galaxy/webapps/galaxy/api/library_contents.py` , `lib/galaxy/webapps/galaxy/api/roles.py` , `lib/galaxy/webapps/galaxy/api/workflows.py` , `lib/galaxy/webapps/galaxy/controllers/dataset.py`